auacm/app/modules/submission_manager/judge.py

```diff
@@ -0,0 +1,225 @@
+import itertools
+import os
+import os.path
+import shlex
+import subprocess
+import threading
+import time
+
+from app import app
+
+
+ALLOWED_EXTENSIONS = ['java', 'c', 'cpp', 'py', 'go']
+
+COMPILE_COMMAND = {
+    "java": "javac {0}.java",
+    "py": None,
+    "c": "gcc {0}.c -o {0}",
+    "cpp": "g++ {0}.cpp -o {0}",
+    "go": "go build -o {0} {0}.go"
+}
+RUN_COMMAND = {
+    "java": "java -cp {0}/ {1}",
+    "py": "python {0}/{1}.py",
+    "c": "{0}/{1}",
+    "cpp": "{0}/{1}",
+    "go": "{0}/{1}"
+}
+TIMEOUT_MULTIPLIER = {
```

Comment:
This works for now, but we've talked about doing empirical tests to determine multipliers